src/Utilities.UnitTests/TrackedDependencies/FileTrackerTests.cs

```diff
@@ -282,6 +286,8 @@ public void FileTrackerFindStrInOperationsExtended_AttributesOnly()
             Assert.True(foundCreateFileW || foundCreateFileA);
         }
 
+
+#if ENABLE_TRACKER_TESTS // https://github.com/dotnet/msbuild/issues/79122
```

Comment:
This is too big a number for us :)

Reply:
Created accidentally in roslyn repo, also noticed :)